go.mod

```diff
@@ -3,7 +3,6 @@ module kubevirt-csi-driver-operator
 go 1.19
```

Comment:
it's my fault, but could you change `module kubevirt-csi-driver-operator` to
`module github.com/kubermatic/kubevirt-csi-driver-operator`?